babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -351,10 +351,46 @@ SqlCreateAttribute CreateTableAttributeFreeSpace() :
         }
     }
     [ <PERCENT> { percent = true; } ]
-    { return new SqlCreateAttributeFreeSpace(freeSpaceValue, percent, getPos()); }
+    { return new SqlTableAttributeFreeSpace(freeSpaceValue, percent, getPos()); }
 }
 
-SqlCreateAttribute CreateTableAttributeIsolatedLoading() :
+/**
+ * Parses FREESPACE attribute in ALTER TABLE queries.
+ * Can either specify a value, or DEFAULT FREESPACE.
+ */
+SqlTableAttribute AlterTableAttributeFreeSpace() :
+{
+    SqlLiteral tempNumeric;
+    int freeSpaceValue;
+    boolean percent = false;
+    boolean isDefault = false;
+}
+{
+    (
```

Comment:
nit: Unnecessary parentheses at line 369,386

Reply:
They are not unnecessary - there is a | at 380 separating two cases.

Reply:
Reasonable